script.js

```diff
@@ -0,0 +1,245 @@
+// eng key lang
+
+const keyboardEng = [
+  ['`~', '1!', '2@', '3#', '4$', '5%', '6^', '7&', '8*', '9(', '0)', '-_', '=+',
+    {
+      key: 'Backspace',
+      name: 'backsp'
+    }
+  ],
+  [{
+    key: 'Tab',
+    name: 'tab'
+  }, 'qQ', 'wW', 'eE', 'rR', 'tT', 'yY', 'uU', 'iI', 'oO', 'pP', '[{', ']}', '\/|'],
+  [{
+    key: 'CapsLock',
+    name: 'caps'
+  }, 'aA', 'sS', 'dD', 'fF', 'gG', 'hH', 'jJ', 'kK', 'lL', ';:', "'\"", {
+    key: 'Enter',
+    name: 'enter'
+  }],
+  [{
+    key: 'Shift',
+    name: 'shift'
+  }, 'zZ', 'xX', 'cC', 'vV', 'bB', 'nN', 'mM', ',<', '.>', '/?', {
+    key: 'Shift',
+    name: 'shift'
+  }],
+  [{
+    key: 'Control',
+    name: 'ctrl'
+  }, {
+    key: 'F1',
+    name: 'fn'
+  }, {
+    key: 'Alt',
+    name: 'alt'
+  }, {
+    key: ' ',
+    name: 'space'
+  }, {
+    key: 'Alt',
+    name: 'alt'
+  }, {
+    key: 'F1',
+    name: 'fn'
+  }, {
+    key: 'Control',
+    name: 'ctrl'
+  }]
+];
+
+// rus key lang
+
+const keyboardRus = [
+  ['ёЁ', '1!', '2"', '3№', '4;', '5%', '6:', '7?', '8*', '9(', '0)', '-_', '=+', 'backsp'],
+  ['tab', 'йЙ', 'цЦ', 'уУ', 'кК', 'еЕ', 'нН', 'гГ', 'шШ', 'щЩ', 'зЗ', 'хХ', 'ъЪ', '\//'],
+  ['caps', 'фФ', 'ыЫ', 'вВ', 'аА', 'пП', 'рР', 'оО', 'лЛ', 'дД', 'жЖ', "эЭ", 'enter'],
+  ['shift', 'яЯ', 'чЧ', 'сС', 'мМ', 'иИ', 'тТ', 'ьЬ', 'бБ', 'юЮ', '.,', 'shift'],
+  ['ctrl', 'fun', 'alt', 'space', 'alt', 'fun', 'ctrl']
+];
+
+let keyboard = keyboardEng;
+const div = document.createElement('div');
```

Comment:
Please be more explicit in your variables name. `div` is just something abstract, even `wrapper` would be better, not perfect though